docs/curl.1

```diff
@@ -150,6 +150,10 @@ version. (Added in 7.33.0)
 .IP "--http2"
 (HTTP) Tells curl to issue its requests using HTTP 2. This requires that the
 underlying libcurl was built to support it. (Added in 7.33.0)
+.IP "--http2-no-upgrade"
+(HTTP) Tells curl to issue its requests using HTTP 2 over clean TCP without 
+HTTP/1.1 Upgrade. This requires that the underlying libcurl was built to 
+support it. (Added in 7.49.0)
```

Comment:
I see 2 options:
-- We can ignore this option if there is any output from ALPN ( current implementation )
-- Ignore this option on HTTPS ( not reflected by the current code )

Other idea is not to implement this option as a VERSION but as a new curl-option for the request.